getting-started/pull-request-lifecycle.rst

```diff
@@ -183,59 +183,93 @@ message. It is usually okay to leave that as-is and close the editor.
 See `the merge command's documentation <https://git-scm.com/docs/git-merge>`_
 for a detailed technical explanation.
 
-
 .. _good-prs:
 
-Making good PRs
+Making Good PRs
```

Comment:
Please retain sentence case: https://devguide.python.org/documentation/style-guide/#capitalization
```suggestion
Making good PRs
```
Same applies to the headers below.